Gir.toml

```diff
@@ -1,33 +1,43 @@
 [options]
-gir_dir = "input"
+girs_dir = "gir-files"
 library = "Vte"
 version = "2.91"
 work_mode = "normal"
-generate_safety_asserts = true
 external_libraries = [
     "Gdk",
     "Gio",
     "GLib",
     "Gtk",
     "Pango",
 ]
+target_path = "."
+generate_safety_asserts = true
+deprecate_by_min_version = true
+single_version_file = true
 
 generate = [
     "Vte.CursorBlinkMode",
     "Vte.CursorShape",
     "Vte.EraseBinding",
-    "Vte.Pty",
-    "Vte.PtyError",
-    "Vte.PtyFlags",
+#    "Vte.Pty",
+#    "Vte.PtyError",
+#    "Vte.PtyFlags",
     "Vte.Regex",
     "Vte.SelectionFunc",
-    "Vte.Terminal",
     "Vte.WriteFlags",
 ]
 
 manual = [
     "Gdk.CursorType",
+    "Gdk.RGBA",
     "GLib.Error",
     "Gtk.Widget",
     "Pango.FontDescription",
 ]
+
+[[object]]
+name = "Vte.Terminal"
+status = "generate"
+    [[object.function]]
+    name = "set_colors"
+    ignore = true
```

Comment:
This function isn't generated properly, it tries to pass a gdk::RGBA instead of gdk_sys::GdkRGBA into the function it calls.